.github/workflows/test.yml

```diff
@@ -123,6 +123,7 @@ jobs:
       - uses: actions/checkout@v4
       - uses: ./
         with:
+          micromamba-version: 2.0.2-0
```

Comment:
This is temporary and can be reverted once this PR has been merged and a new version of `setup-micromamba` has been released.

Reply:
No, this is hiding the test. We are testing with micromamba 2.0.2 instead of micromamba 2.0.4